en/backup-to-aws-s3-by-snapshot.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Back Up a TiDB Cluster Using EBS Volume Snapshots
+summary: Learn how to back up TiDB cluster data to S3 based on EBS volume snapshots using TiDB Operator.
+---
+
+# Back Up a TiDB Cluster Using EBS Volume Snapshots
+
+> **Warning:**
+>
+> This feature is still experimental. It is NOT recommended that you use it in the production environment.
+
+This document describes how to back up a TiDB cluster deployed on AWS Elastic Block Store (EBS) in Kubernetes to AWS S3.
+
+The backup method described in this document is implemented based on CustomResourceDefinition (CRD) in TiDB Operator. For the underlying implementation, [BR](https://docs.pingcap.com/tidb/stable/backup-and-restore-overview) is used to get the backup data of the TiDB cluster, and then send the data to the AWS storage. BR stands for Backup & Restore, which is a command-line tool for distributed backup and recovery of the TiDB cluster data. If your TiDB cluster is deployed on AWS EKS and uses EBS volumes, you can use the method described in this document to back up the TiDB cluster.
+
+## Recommended scenarios and limitations
+
+### Recommended scenarios
+
+If you have the following requirements when backing up TiDB cluster data, you can use TiDB Operator to back up the data by volume snapshots and metadata to AWS S3:
+
+- Minimize the impact of backup, for example, to keep the impact on QPS and transaction latency less than 5%, and to occupy no cluster CPU and memory.
+- Back up and restore data in a short time. For example, finish backup within 1 hour and restore in 2 hours.
+
+If you have any other requirements, select an appropriate backup method based on [Backup and Restore Overview](backup-restore-overview.md).
+
+### Limitations
+
+- Snapshot backup is applicable to TiDB Operator v1.4.0 or above, and TiDB v6.3.0 or above.
+- To use this backup method, the TiDB cluster must be deployed on AWS EKS and uses AWS EBS volumes.
```

Comment:
According to L12, "This document describes how to back up a TiDB cluster deployed on AWS Elastic Block Store (EBS) in Kubernetes to AWS S3.", so the TiDB cluster can be deployed on AWS EBS as well?

Reply:
@fengou1 PTAL

Reply:
Fixed in [d38c0c4](https://github.com/pingcap/docs-tidb-operator/pull/2065/commits/d38c0c435e70549e07be0fdb36b73645dac4589b)